pkgs/development/compilers/ghc/common-hadrian.nix

```
@@ -271,8 +259,20 @@
         })
       ]
 
-      # Missing ELF symbols
-      ++ lib.optionals stdenv.targetPlatform.isAndroid [
+      # Unconditionally applied while we wait for upstream developments
+      # (issue/fix/merge/release) to inform specific guards
```

```suggestion
      # Unconditionally applied while we wait for upstream developments
      # (issue/fix/merge/release) to inform specific guards.
```

I'm not sure what is meant with "to inform specific guards".